### DISCUSSION TITLE
When are redirects processed? Build-time only? How can we fetch them from a CMS?

### DISCUSSION THREAD

**Discussion by Drarok** [vercel/next.js#32469] [Help] [answered]
During development, Next.js works great at fetching redirects using the `redirects` async function in `next.config.js`.

Having built and deployed to production, none of the changes in the CMS are appearing, so I'm trying to confirm if redirects are only processed _once,_ at build-time, or if there's an option I need to turn on to enable a more ISR-like approach with the defaults from the build, but also occasional updates from the CMS?

**Comment by leerob** [comment, marked as answer]
Correct, redirects are processed at build time.

Some potential solutions here -> https://vercel.com/support/articles/how-can-i-increase-the-limit-of-redirects-or-use-dynamic-redirects-on-vercel
